src/components/sendV2/summary/summary.js

```diff
@@ -181,7 +240,7 @@ class Summary extends React.Component {
             className={`${styles.btn} on-nextStep`}
             onClick={this.nextStep}
             disabled={isBtnDisabled}>
-            {this.props.t('Send {{value}} LSK', { value: this.props.fields.amount.value })}
+            {this.props.t('{{value}}', { value: `${confirmBtnMessage}` })}
```

Comment:
With the above suggestions you would not need the `this.props.t` here.

Reply:
done